pokemon_v2/urls.py

```diff
@@ -63,7 +63,9 @@
 router.register(r"version", VersionResource)
 router.register(r"version-group", VersionGroupResource)
 router.register(r"honey-trees", HoneyTreeResource)
-
+router.register(
+    r"Trophy_Garden_Special_Encounters", TrophyGardenSpecialEncounterResource
```

Comment:
Please rename to be consistent with other URLs
```suggestion
    r"encounter-trophy-garden", TrophyGardenSpecialEncounterResource
```